distributed-coordinator/src/main/java/com/gentics/mesh/distributed/coordinator/MasterElector.java

```diff
@@ -153,7 +158,9 @@ private void electMaster() {
 				if (clusterOptions.getCoordinatorTopology() == CoordinationTopology.MASTER_REPLICA) {
 					database.setToMaster();
 				}
-				localMember().setBooleanAttribute(MASTER, true);
+				MeshMemberInfo info = members.get(localMember().getUuid());
+				info.isMaster = true;
```

Comment:
Yes, as soon the instance is properly started. Otherwise it won't be able to serve properly anyway. One can wrap a NPE into more meaningful message though.